devices/xiaomi.js

```diff
@@ -1381,6 +1381,24 @@ module.exports = [
         },
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['lumi.curtain.acn003'],
+        model: 'ZNCLBL01LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara curtain motor E1',
```

Comment:
I also suggest changing the device name to `Aqara curtain driver E1` as it is indicated in the _Aqara Home_ app:

![ZNCLBL01LM](https://user-images.githubusercontent.com/38351800/171011227-02726f3f-f9b3-4d71-b8a4-34a56ce9105e.png)